releases/fuji/compose-files/docker-compose-fuji-1.0.0.yml

```diff
@@ -0,0 +1,579 @@
+            - edgex-files
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+      - secrets-setup-cache:/etc/edgex/pki
+
+  consul:
+    image: consul:1.3.1
+    ports:
+      - "8400:8400"
+      - "8500:8500"
+      - "8600:8600"
+    container_name: edgex-core-consul
+    hostname: edgex-core-consul
+    networks:
+      edgex-network:
+        aliases:
+            - edgex-core-consul
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - volume
+
+  config-seed:
+    image: edgexfoundry/docker-core-config-seed-go:1.0.0
+    container_name: edgex-config-seed
+    hostname: edgex-core-config-seed
+    networks:
+      edgex-network:
+        aliases:
+            - edgex-core-config-seed
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - volume
+      - consul
+
+  vault:
+    image: edgexfoundry/docker-edgex-vault:1.1.0-dev
+    container_name: edgex-vault
+    hostname: edgex-vault
+    networks:
+      edgex-network:
+        aliases:
+            - edgex-vault
+    ports:
+      - "8200:8200"
+    cap_add:
+      - "IPC_LOCK"
+    environment:
+      - VAULT_ADDR=https://edgex-vault:8200
+      - |
+        VAULT_LOCAL_CONFIG=
+          listener "tcp" { 
+              tls_disable = "0" 
+              cluster_address = "edgex-vault:8201" 
+              tls_min_version = "tls12" 
+              tls_client_ca_file ="/run/edgex/secrets/edgex-vault/ca.pem" 
+              tls_cert_file ="/run/edgex/secrets/edgex-vault/server.crt" 
+              tls_key_file = "/run/edgex/secrets/edgex-vault/server.key" 
+              tls_perfer_server_cipher_suites = true 
+          } 
+          backend "consul" { 
+              path = "vault/" 
+              address = "edgex-core-consul:8500" 
+              scheme = "http" 
+              redirect_addr = "https://edgex-vault:8200" 
+              cluster_addr = "https://edgex-vault:8201" 
+          } 
+          default_lease_ttl = "168h" 
+          max_lease_ttl = "720h"
+      - VAULT_CONFIG_DIR=/vault/config
+      - VAULT_UI=true
+    tmpfs:
+      - /run
+    command: >
+      /usr/bin/dumb-init -- /bin/sh -c 
+      "./security-secrets-setup generate ; 
+      chown -Rh 100:1000 /run/edgex/secrets/edgex-vault/ ;
+      while test \! -f /run/edgex/secrets/edgex-vault/.security-secrets-setup.complete; do
+        sleep 1;
+      done;
+      exec /usr/local/bin/docker-entrypoint.sh server -log-level=info"
+    volumes:
+      - vault-file:/vault/file
+      - vault-logs:/vault/logs
+      - secrets-setup-cache:/etc/edgex/pki
+      - /run/edgex/secrets/edgex-vault:/run/edgex/secrets/edgex-vault
+    depends_on:
+      - volume
+      - consul
+
+  vault-worker:
```

Comment:
since we don't have the other new Fuji containers yet and those are missing from this file AFAICT, we should probably drop this old Edinburgh container from the YAML too, no?

Reply:
@anonymouse64 Ian, I thought we still need the vault worker to init and unseal Vault; understanding that the init-resp.json for Vault's token still not encrypted yet but it is supposed to work.  Correct me if I am wrong, @tingyuz and @bnevis-i

Reply:
The container and function it still valid -- the underlying executable has been renamed to security-secretstore-setup instead of vault-worker.  It may have devops impact to rename the container.

Reply:
hmm I thought we renamed the docker container too, but if we didn't then this is fine I understand the need for it, I just thought that :
* we were going to rename the docker container
* we weren't including the other docker containers here because they hadn't been renamed yet

which made me think this was left in by mistake

Reply:
I can echo that we decided to change the names of the underlying executable but no the names of the containers. Given the limit time I would like to keep the original container names but add this into backlog.

Reply:
filed as https://github.com/edgexfoundry/edgex-go/issues/1933